### PR TITLE
Docs redirects

### PR DESCRIPTION
**Pull request by Nadine2016**
# Description

## Docs changes

# Context

## Notes for reviewers

# Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: docs/build-docs.sh [hunk 1]
```diff
@@ -289,7 +289,12 @@ firebaseJson=$(cat <<EOF
         "source": "/gloo-edge/:version/api/github.com/solo-io/gloo/projects/gloo/api/v1/proxy.proto.sk/",
         "destination": "/gloo-edge/:version/reference/api/github.com/solo-io/gloo/projects/gloo/api/v1/proxy.proto.sk/",
         "type":"301"
-      }
+      },
+      {
+        "source": "/gloo-edge/v1.13.xreference/api/github.com/solo-io/gloo/projects/gloo/api/v1/options/tracing/tracing.proto.sk/",
```
Comment (Rachael-Graham) [inline docs/build-docs.sh:294]:
```suggestion
        "source": "/gloo-edge/v1.13.x/reference/api/github.com/solo-io/gloo/projects/gloo/api/v1/options/tracing/tracing.proto.sk/",
```

Comment (artberger) [inline docs/build-docs.sh:294]: is this missing a backslash?
> v1.13.xreference

Comment (Nadine2016) [reply docs/build-docs.sh:294]: no it actually is like this. haha

**Review by artberger — APPROVED** [review]

**Pull request merged**